attributes/default.rb

```diff
@@ -39,6 +47,9 @@
 when 'openbsd', 'freebsd', 'mac_os_x'
   default['ssl_certificate']['key_dir'] = '/etc/ssl'
   default['ssl_certificate']['cert_dir'] = '/etc/ssl'
+when 'windows'
+  default['ssl_certificate']['key_dir'] = Chef::Config[:file_cache_path]
+  default['ssl_certificate']['cert_dir'] = Chef::Config[:file_cache_path]
```

Comment:
I'm not 100% sure here, but I think using that *Certlog* directory sounds better than using an erasable cache directory.